kDriveCore/Data/Cache/DriveFileManager.swift

```diff
@@ -69,6 +83,11 @@ public class DriveFileManager {
                 if oldSchemaVersion < 13 {
                     migration.deleteData(forType: UploadFile.className())
                 }
+
+                // Migration for Upload APIV2 build3
+                if oldSchemaVersion < 14 {
```

Comment:
Since we are doing the same for 13 and 14 we can simply remove the block for 13

Reply:
nice catch